lib/service/service.go

```diff
@@ -728,6 +733,8 @@ func (process *TeleportProcess) initAuthService() error {
 	})
 
 	process.RegisterFunc("auth.tls", func() error {
+		utils.Consolef(cfg.Console, "[AUTH]  Auth service is starting on %v.", cfg.Auth.SSHAddr.Addr)
```

Comment:
These log lines will be misaligned because they don't use the new component logger we have.

Should be easy to fix because `utils.Consolf` jus writes to a writer and then to the logs: https://github.com/gravitational/teleport/blob/v2.5.2/lib/utils/cli.go#L140-L148